benchmarks/imports.py

```diff
@@ -15,3 +15,15 @@ def time_imports(self):
         repo = f"file://{self.project_dir}"
         path = "data/cats_dogs"
         self.dvc("import", repo, path, proc=True)
+
+
+class ImportUrlToRemoteBench(BaseRemoteBench):
+    repeat = 1
+    timeout = 12000
+
+    def setup(self):
+        super().setup()
+        self.data_url = self._remote_prefix + self.setup_data("mini")
+
+    def time_import_url_to_remote(self):
+        self.dvc("import-url", self.data_url, "--to-remote")
```

Comment:
Lets use `proc=True` here. Usually we want to use `proc=True` in order to not have "pre-imported" `dvc` modules. I wonder whether we should not have a special method to call in `time` methods that would make sure `proc` is on by default. Its easy to omit, even during review.